setup.py

```diff
@@ -166,17 +169,20 @@ def _build_win_compression(self):
     def run(self):
         # First, we build the standard extensions.
         _build_ext.run(self)
+
         # Then, we build the compression tools and the driver if required
-        driver_build_function = lambda *args: None
```

Comment:
Are lambda functions against flake8?

Reply:
Assignment of lambda functions is what Flake 8 is complaining about.  They are really meant to be used inline.